js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/getEvaluationsWithEmptyAdded.tsx

```diff
@@ -15,34 +15,46 @@ export const getEvaluationsWithEmptyAdded = ({
   isFirstPage,
   isLastPage,
 }: Config) => {
-  if (isLoading || !currentEvaluationId) {
+  if (isLoading) {
     return [];
   }
+
   const evalsWithSkips = [];
-  let current = isFirstPage ? currentEvaluationId : evaluations[0]?.evaluationId || 1;
-  evaluations.forEach((evaluation, i) => {
-    const prevEvaluation = evaluations[i - 1];
-    if (evaluation.evaluationId !== current) {
-      evalsWithSkips.push({
-        __typename: 'no_conditions_met' as const,
-        evaluationId: current,
-        amount: current - evaluation.evaluationId,
-        endTimestamp: prevEvaluation?.timestamp ? prevEvaluation?.timestamp - 60 : ('now' as const),
-        startTimestamp: evaluation.timestamp + 60,
-      });
-    }
-    evalsWithSkips.push(evaluation);
-    current = evaluation.evaluationId - 1;
-  });
-  if (isLastPage && current > 0) {
+
+  if (evaluations.length) {
```

Comment:
Do we need an `evaluations.length` condition, since if the array is empty then `evaluations.forEach` will no-op, achieving the same thing?